v_4_0_0/files/k8s-resource/calico-all.yaml

```diff
@@ -215,7 +215,7 @@ spec:
               memory: 150Mi
             limits:
               cpu: 250m
-              memory: 150Mi
+              memory: 500Mi
```

Comment:
Do we now have consensus on keeping requests (for scheduling) as-is but moving limit to `500MB`? Before merge I'll then figure out which ones are actually needed for required patch releases and copy those.

Reply:
I thought requests: 250 and limit almost unlimited. Where does the 150 and 500 come from now?

Reply:
`150MB` is the current limit which *most* the time is perfectly fine. It's only occasional peak that runs above it. `500MB` is ~233% more than what is normally enough so for me this is a lot. I don't understand reasoning from something like e.g. `4GB` limit for component like this. What purpose would it have?

Reply:
The purpose is that we don't want calico to get killed. If there is a short spike for whatever reason calico should not get killed.

If the calico pods stays under 150MB for 99% of the time I am fine with that. If we now are more often between 150MB and 250MB then we should consider raising requests as this prevents eviction.